Library/Formula/grafana.rb

```diff
@@ -0,0 +1,92 @@
+require "language/go"
+
+class Grafana < Formula
+  homepage "http://www.grafana.org/"
+  head "https://github.com/grafana/grafana.git"
+  version "2.0.0-beta1"
+
+  stable do
+    url "https://github.com/grafana/grafana/archive/v2.0.0-beta1.tar.gz"
```

Comment:
For Grafana 2.0 there currently is no stable release (latest is beta1). Grafana 2.0 is a substantial update compared to 1.x (new Go backend, etc). Can there be non stable homebrew packages?